tests/openfhe/end_to_end/simple_sum.mlir

```diff
@@ -0,0 +1,11 @@
+// heir-opt --mlir-to-openfhe-bgv='entry-function=simple_sum ciphertext-degree=32' %s | bazel-bin/tools/heir-translate --emit-openfhe-pke
```

Comment:
This seems like a documentation (and not a RUN line for a lit test) since the build target contains the flag info and heir-translate commands. In case the flags change (which I suspect ciphertext-degree may at some point be changed), could you add a comment saying something like "the build target executes the following"? At first glance I assumed this was the RUN line.

Reply:
+1 yeah I left it there as documentation

Reply:
Fixed in 30cdb032